src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -272,6 +272,28 @@ namespace winrt::TerminalApp::implementation
         args.Handled(true);
     }
 
+    void TerminalPage::_HandleSetColorScheme(const IInspectable& /*sender*/,
+                                             const TerminalApp::ActionEventArgs& args)
+    {
+        if (const auto& realArgs = args.ActionArgs().try_as<TerminalApp::SetColorSchemeArgs>())
+        {
+            if (auto activeTab = _GetFocusedTab())
+            {
+                if (auto activeControl = activeTab->GetActiveTerminalControl())
+                {
+                    // Theoretically, if the properties in an IControlSettings

```

Comment:
So is this a TODO or just a musing about something that is too much work to bother doing?

Reply:
musing i think

Reply:
Definitely just musing. I'll yank it.